docker-compose.traefik.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.3"
```

Comment:
Could we use a newer version, like 3.7?

Reply:
From my perspective unless moving to swarm the syntax seems the same.

Reply:
Awesome, so we should move up to alleviate the tech debt?